extensions/nginx/index.js

```diff
@@ -132,23 +132,24 @@ class NginxExtension extends cli.Extension {
         }, {
             title: 'Preparing nginx for SSL configuration',
             task: (ctx) => {
-                return argv.sslemail ? Promise.resolve({email: argv.sslemail}) : this.ui.prompt({
+                let promise = argv.sslemail ? Promise.resolve(argv.sslemail) : this.ui.prompt({
                     name: 'email',
                     type: 'input',
                     message: 'Enter your email (used for SSL certificate generation)',
                     validate: value => Boolean(value) || 'You must supply an email'
-                }).then((answer) => {
-                    argv.sslemail = answer.email;
+                }).then(answer => { argv.sslemail = answer.email; });
```

Comment:
It took me many reads of this to understand that the problem is the then is attached to or clause.

I think it might be worth expanding this out?